pkg/events/core_arm64.go

```diff
@@ -1,5 +1,4 @@
 //go:build arm64
-// +build arm64
```

Comment:
I don't think we have syscall tests in the e2es. We do have them in integration, but integration only runs in amd64. I understand that revive considers this redundant but this is a sensitive change.

So I think we should:
1. Test syscalls manually in this PR in an arm64 machine
2. Add a mirror integration test job that runs on an arm64 runner. I have opened #4621 for this.

Reply:
@rscampos would you mind testing this PR on arm64?

Reply:
tested before and after this PR on arm64 and seems not to have any side effect